### PR TITLE
[sw, rom_ext_signer] use mundane exponent and modulus getters instead of dummies

### PR DESCRIPTION
**Pull request by silvestrst**
This change also updates the documentation.

### CODE REVIEW

File: sw/host/rom_ext_image_tools/signer/src/main.rs [hunk 1]
```diff
@@ -40,16 +40,18 @@ fn main() {
...
 
-    let exponent = &signature_key_public_exponent_le();
-    image.update_exponent_field(exponent);
-
-    let modulus = &signature_key_modulus_le();
-    image.update_modulus_field(modulus);
-
     // Convert ASN.1 DER private key into Mundane RsaPrivKey.
     let private_key =
         RsaPrivKey::parse_from_der(&private_key_der).expect("Failed to parse private key!");
```
Comment (mundaym) [inline sw/host/rom_ext_image_tools/signer/src/main.rs:46]: Maybe add a comment like:

```rust
// Encode public key components in little-endian byte order.
```

Just to hint why the `reverse()` is needed.

**Review by mundaym — APPROVED** [review]
LGTM

**Review by alphan — APPROVED** [review]

**Review by moidx — APPROVED** [review]
LGTM

File: sw/host/rom_ext_image_tools/signer/src/main.rs (hunk 1 above)
Comment (silvestrst) [reply sw/host/rom_ext_image_tools/signer/src/main.rs:46]: Done.

**Pull request merged**